src/builder.rs

```diff
@@ -568,11 +568,28 @@ impl<'a, 'gcc, 'tcx> BuilderMethods<'a, 'tcx> for Builder<'a, 'gcc, 'tcx> {
     ) {
         let mut gcc_cases = vec![];
         let typ = self.val_ty(value);
-        for (on_val, dest) in cases {
-            let on_val = self.const_uint_big(typ, on_val);
-            gcc_cases.push(self.context.new_case(on_val, on_val, dest));
+        // FIXME: This is a workaround for a libgccjit limitation.
```

Comment:
It is preferred in the Rust project to specify a username, so please use something like this:
```suggestion
        // FIXME(antoyo): This is a workaround for a libgccjit limitation.
```